hello.ahk

```diff
@@ -1,11 +1,11 @@
-﻿SendMode Input  ; Recommended for new scripts due to its superior speed and reliability.
+SendMode Input  ; Recommended for new scripts due to its superior speed and reliability.
 SetWorkingDir %A_ScriptDir%  ; Ensures a consistent starting directory.
 #singleinstance force
 
 #Persistent
 #InstallKeybdHook
 #InstallMouseHook
-#Include ..\Private_Folder\KeyLogLib.ahk
```

Comment:
This doesn't seem to have correct parameter for it. I recommend putting the Private_Folder with the scripts inside the same directory with the hello.ahk, and then
	#Include ..\Private_Folder\KeyLogLib.ahk
	; should be:
	#Include %A_ScriptDir%\Private_Folder\KeyLogLib.ahk